src/viewProviders/flinkArtifacts.test.ts

```diff
@@ -1,12 +1,14 @@
 import * as assert from "assert";
 import * as sinon from "sinon";
 import { CancellationToken, Progress, window } from "vscode";
+import { ResponseError } from "vscode-languageclient";
```

Comment:
Make sure this is imported from https://github.com/confluentinc/vscode/tree/main/src/clients/flinkArtifacts since `isErrorResponse` only keeps track of specific ResponseErrors: https://github.com/confluentinc/vscode/blob/8468f8419d03699bb3b1e9bd7a630d2e29588800/src/errors.ts#L34-L45